bin/ops/purge_backups.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
Is this file the same as `backend/br_ops/purge_backups`? Does it need to be duplicated here for some reason?

Reply:
Yes, the script in bin/ops needs to be removed